example/lib/main.dart

```diff
@@ -2,16 +2,19 @@ import 'package:feature_discovery/feature_discovery.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/scheduler.dart';
 
-const String feature1 = "FEATURE_1";
-const String feature2 = "FEATURE_2";
-const String feature3 = "FEATURE_3";
-const String feature4 = "FEATURE_4";
-const String feature5 = "FEATURE_5";
-const String feature6 = "FEATURE_6";
-const String feature7 = "FEATURE_7";
+const String feature1 = 'feature1',
+    feature2 = 'feature2',
+    feature3 = 'feature3',
+    feature4 = 'feature4',
+    feature5 = 'feature5',
+    feature6 = 'feature6',
+    feature7 = 'feature7';
```

Comment:
You taught me something. I was totally unaware of this Dart 'functionality'

Reply:
Maybe it looks a bit magical because of the formatting, but it is just the same as `int a = 1, b = 2;`, or were you not aware of that 😄

Reply:
I was not 😅

Reply:
It's quite useful to not have to repeat the type every time 👍